content/blog/suva-abredeversicherung/suva-abredeversicherung.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Suva Chatbot für Abredeversicherung
+permalink: /blog/suva-chatbot-fuer-abredeversicherung
+templateKey: blog-post
+image: ./suva-chatbot.png
+date: 2020-03-06T11:16:49
+author: Philip Schönholzer
+description: Seit ein paar Wochen läuft unser Chatbot bei der Suva, um die Besucher beim Abschluss der Abredeversicherung zu unterstützen.
+categories:
+  - Projekte
+---
+
+## Ziele
+
+Die Suva bietet eine temporäre Unfallversicherung für Private, welche für eine Weile nicht Arbeiten. Diese Versicherung nennt die Suva Abredeversicherung. Wie in der Versicherungsbranche leider üblich, ist man als Kunde nicht sicher, wie die Versicherung funktioniert, was die Voraussetzung für diese sind, und welche Leistungen gedeckt sind.
```

Comment:
Die Suva bietet eine temporäre Unfallversicherung für Private an, welche für eine Weile nicht arbeiten. Es handelt sich dabei um eine sogenannte Abredeversicherung. Oft kommt es in der Versicherungsbranche vor, dass man sich als Kunde nicht sicher ist, wie die Versicherung funktioniert, was die Voraussetzung für diese sind, und welche Leistungen gedeckt sind. Genau diese Unsicherheiten soll der Chatbot adressieren.